test/battle/battle.test.ts

```diff
@@ -277,7 +277,7 @@ describe("Test Battle Phase", () => {
     game.override.startingWave(3);
     game.override.moveset([moveToUse]);
     game.override.enemyMoveset([MoveId.TACKLE, MoveId.TACKLE, MoveId.TACKLE, MoveId.TACKLE]);
-    await game.classicMode.startBattle([SpeciesId.DARMANITAN, SpeciesId.CHARIZARD]);
+    await game.classicMode.startBattle(SpeciesId.DARMANITAN, SpeciesId.CHARIZARD);
```

Comment:
```suggestion
    await game.classicMode.startBattle(SpeciesId.MEWTWO, SpeciesId.CHARIZARD);
```
again (since the starter species override only applies to the first mon, this ensure the same behavior as before)
TBH the species here probably don't matter but I'm worried this mix and match of ways to override the starter species could cause issues